ibm_catalog.json

```diff
@@ -68,33 +68,39 @@
                   }
                 ]
               },
-              {
-                "key": "use_existing_resource_group"
-              },
               {
                 "key": "resource_group_name",
                 "required": true
               },
+              {
+                "key": "use_existing_resource_group"
```

Comment:
as per this [issue](https://github.ibm.com/GoldenEye/issues/issues/12542), you should force this to true

Reply:
Updated to be required in catalog json